locations/spiders/softbank_jp.py

```
@@ -0,0 +1,66 @@
+        return item
+
+    @staticmethod
+    def parse_hours(hours: dict) -> OpeningHours:
+        opening_hours = OpeningHours()
+        for day, times in hours.items():
+            # The weekdays are joined by a "holiday" entry for public holidays, which
+            # sanitise_day returns None for because OpeningHours has no day to put it on.
+            if not (weekday := sanitise_day(day)):
+                continue
+            # `reception_time` beside it is a 受付時間 desk cut-off, not the shop's own hours.
+            opening, _, closing = str((times or {}).get("business_hours") or "").partition("～")
```

Ruff (0.16.3)

[warning] String contains ambiguous `～` (FULLWIDTH TILDE). Did you mean `~` (TILDE)?

(RUF001)